tests/test_namelist.py

```diff
@@ -182,8 +184,8 @@ def test_enable_restart_from_default(self):
             restart_namelist_settings, restart_config["namelist"]
         )
 
-    def test_enable_restart_from_empty_fv_core_nml(self):
-        restart_config = enable_restart(config_with_empty_fv_core_nml)
+    def test_enable_restart_from_empty_fv_core_and_coupler_nml(self):
+        restart_config = enable_restart(config_with_empty_fv_core_and_coupler_nml)
```

Comment:
Should this work on an entirely empty namelist? I thought there's no difference from a Fortran standpoint between not defining one of the namelists or defining it to be empty? If we do want to support empty namelists (e.g. where coupler_nml is not present in the `.nml` file at all) please add a test case.

Reply:
Okay, I allowed the namelist to not contain `fv_core_nml` or `coupler_nml` but the config object still has to have a `namelist` item, which I believe is a requirement in various other places of this package.